### PR TITLE
Data scientist scripts

### PR DESCRIPTION
**Pull request by nkashy1**
Simple scripts for data scientist workflows.

### CODE REVIEW

File: scripts/create-task.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    echo "ERROR: Please pass hyperparameters name as the second argument to this script"
+    exit 1
+fi
+
+if [ -z $CHECKPOINT ]; then
+    echo "ERROR: Please pass checkpoint name as the third argument to this script"
+    exit 1
+fi
+
+if [ -z $LINK ]; then
```
Comment (tpdocai) [inline scripts/create-task.sh]: This should be task here and in the message

File: scripts/create-task.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    echo "ERROR: Please pass task name as the fourth argument to this script"
+    exit 1
+fi
+
+if [ -z $LINK ]; then
+    echo "ERROR: Please pass checkpoint link as the fifth argument to this script"
```
Comment (tpdocai) [inline scripts/create-task.sh]: Should be link in the message.

File: scripts/create-checkpoint.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    exit 1
+fi
+
+if [ -z $CHECKPOINT ]; then
+    echo "ERROR: Please pass checkpoint name as the third argument to this script"
+    exit 1
+fi
+
+if [ -z $LINK ]; then
+    echo "ERROR: Please pass checkpoint link as the fourth argument to this script"
```
Comment (tpdocai) [inline scripts/create-checkpoint.sh]: link in the message

File: scripts/create-model.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+REPOSITORY_API_URL=${REPOSITORY_API_URL:-localhost:8081}
+REPOSITORY_API_TOKEN=${REPOSITORY_API_TOKEN:-WriterToken}
+MODEL=$1
+DESCRIPTION=$2
+
+if [ -z $MODEL ]; then
+    echo "ERROR: Please pass model name as the first argument to this script"
+    exit 1
+fi
+
```
Comment (tpdocai) [inline scripts/create-model.sh]: Do you need to check for $DESCRIPTION? I guess empty is Ok.

Comment (nkashy1) [reply scripts/create-model.sh]: Yep, empty is ok.

File: scripts/create-checkpoint.sh (hunk 3 above)
Comment (nkashy1) [reply scripts/create-checkpoint.sh]: "checkpoint link". Works?

File: scripts/create-task.sh (hunk 2 above)
Comment (nkashy1) [reply scripts/create-task.sh]: "checkpoint link". Works?

File: scripts/create-task.sh (hunk 1 above)
Comment (nkashy1) [reply scripts/create-task.sh]: "task name". Works?

File: scripts/create-checkpoint.sh (hunk 3 above)
Comment (tpdocai) [reply scripts/create-checkpoint.sh]: MY bad.

File: scripts/create-task.sh (hunk 2 above)
Comment (tpdocai) [reply scripts/create-task.sh]: Yes

**Pull request merged**